resourcemanager/pooltask/task_manager_iterator.go

```diff
@@ -0,0 +1,128 @@
+// Copyright 2022 PingCAP, Inc.
```

Comment:
```suggestion
// Copyright 2023 PingCAP, Inc.
```

Reply:
Please add UTs for this file.

Reply:
This test is in ```util/gpool/spmc/spmcpool_test.go```.

Reply:
I feel that it is an independent library and should have its own UTs.

Reply:
In fact, it is convenient for us to write a unit test for this at the pool. But I find we cannot see the UT coverage in the PR. I have reported it to EE team.